scripts/harbor-init/harbor-init.go

```diff
@@ -0,0 +1,236 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"flag"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+)
+
+const (
+	// Harbor API version
+	HARBOR_API_ENDPOINT = "/api/v2.0"
+)
+
+type Login struct {
+	Url      string
+	Username string
+	Password string
+}
```

Comment:
suggestion: Same as the password flag. You should set `String()` and `Format()` method here to not print the password. Or you can make the new type I suggested and make `Password` be a `SensitiveString`.